doc/modules/linear_model.rst

```diff
@@ -1088,9 +1099,9 @@ to warm-starting (see :term:`Glossary <warm_start>`).
         "A Blockwise Descent Algorithm for Group-penalized Multiresponse and
         Multinomial Regression." <1311.6529>`
 
-.. _Generalized_linear_regression:
+.. _Generalized_linear_models:
```

Comment:
Maybe we can try to preserve the old reference identifier in order to not break old URLs with that anchor value for links coming from external websites.

```suggestion
.. _Generalized_linear_regression:

.. _Generalized_linear_models:
```

I am not 100% sure this is authorized in sphinx though. Let's try if it works in the rendered HTML on the CI.

Reply:
I tried this. Sphinx does not throw any errors and it works as expected. I will make this change.